components/create_post/create_post.tsx

```diff
@@ -1135,16 +1132,23 @@ class CreatePost extends React.PureComponent<Props, State> {
             return;
         }
 
-        let type;
-        if (lastPost.root_id && lastPost.root_id.length > 0) {
-            type = Utils.localizeMessage('create_post.comment', Posts.MESSAGE_TYPES.COMMENT);
-        } else {
-            type = Utils.localizeMessage('create_post.post', Posts.MESSAGE_TYPES.POST);
-        }
         if (this.textboxRef.current) {
             this.textboxRef.current.blur();
         }
-        this.props.actions.setEditingPost(lastPost.id, 'post_textbox', type);
+
+        let modalTitle;
```

Comment:
Instead of passing the title to the modal, perhaps the modal could work this out itself?

Reply:
✅  Updated as per request/discussion